widget/radio.go

```diff
@@ -164,11 +164,11 @@ func (r *radioRenderer) Destroy() {
 // Changing the selection (only one can be selected) will trigger the changed func.
 type Radio struct {
 	DisableableWidget
-	Options  []string
-	Selected string
-
-	OnChanged  func(string) `json:"-"`
 	Horizontal bool
+	Mandatory  bool
```

Comment:
Isn't it more commonly known as Required?

Reply:
Dunno … I always say “notwendig” 😜. Will rename it.

Reply:
I was worried it was personal preference but when I googled "mandatory input field" every result used the word "required" :)